Data/PuzzleServerContext.cs

```diff
@@ -79,6 +80,7 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<SinglePlayerPuzzleStatePerPlayer>().HasIndex(pspt => new { pspt.PlayerID, pspt.SolvedTime });
             modelBuilder.Entity<SinglePlayerPuzzleUnlockState>().HasKey(state => new { state.PuzzleID });
             modelBuilder.Entity<Room>().HasIndex(room => new { room.EventID, room.Building, room.Number }).IsUnique();
+            modelBuilder.Entity<Message>().HasIndex(message => new { message.ThreadId });
```

Comment:
@morganbr , I believe this is the spot you talked about if I wanted to add an index to speed up queries.

Are there allowed to be more than one index? I see some of the other items with multiple indexes as one object. Does that mean the index would only work as that kind of composite object? Or is the object considered a list of indexes?

Also, can indexes be null?